dev/bench/bench.sh

```diff
@@ -73,25 +73,22 @@ working_dir="$PWD/${bench_dirname}"
 log_dir=$working_dir/logs
 mkdir "$log_dir"
 
-if [ ! -z "$BENCH_DEBUG" ]
-then
-   echo "DEBUG: ocaml -version = $(ocaml -version)"
-   echo "DEBUG: working_dir = $working_dir"
-   echo "DEBUG: new_ocaml_switch = $new_ocaml_switch"
-   echo "DEBUG: new_coq_repository = $new_coq_repository"
-   echo "DEBUG: new_coq_commit = $new_coq_commit"
-   echo "DEBUG: new_coq_opam_archive_git_uri = $new_coq_opam_archive_git_uri"
-   echo "DEBUG: new_coq_opam_archive_git_branch = $new_coq_opam_archive_git_branch"
-   echo "DEBUG: old_ocaml_switch = $old_ocaml_switch"
-   echo "DEBUG: old_coq_repository = $old_coq_repository"
-   echo "DEBUG: old_coq_commit = $old_coq_commit"
-   echo "DEBUG: old_coq_opam_archive_git_uri = $old_coq_opam_archive_git_uri"
-   echo "DEBUG: old_coq_opam_archive_git_branch = $old_coq_opam_archive_git_branch"
-   echo "DEBUG: num_of_iterations = $num_of_iterations"
-   echo "DEBUG: coq_opam_packages = $coq_opam_packages"
-   echo "DEBUG: coq_pr_number = $coq_pr_number"
-   echo "DEBUG: coq_pr_comment_id = $coq_pr_comment_id"
-fi
+echo "DEBUG: ocaml -version = $(ocaml -version)"
+echo "DEBUG: working_dir = $working_dir"
+echo "DEBUG: new_ocaml_switch = $new_ocaml_switch"
+echo "DEBUG: new_coq_repository = $new_coq_repository"
+echo "DEBUG: new_coq_commit = $new_coq_commit"
+echo "DEBUG: new_coq_opam_archive_git_uri = $new_coq_opam_archive_git_uri"
+echo "DEBUG: new_coq_opam_archive_git_branch = $new_coq_opam_archive_git_branch"
+echo "DEBUG: old_ocaml_switch = $old_ocaml_switch"
+echo "DEBUG: old_coq_repository = $old_coq_repository"
+echo "DEBUG: old_coq_commit = $old_coq_commit"
+echo "DEBUG: old_coq_opam_archive_git_uri = $old_coq_opam_archive_git_uri"
+echo "DEBUG: old_coq_opam_archive_git_branch = $old_coq_opam_archive_git_branch"
+echo "DEBUG: num_of_iterations = $num_of_iterations"
+echo "DEBUG: coq_opam_packages = $coq_opam_packages"
+echo "DEBUG: coq_pr_number = $coq_pr_number"
+echo "DEBUG: coq_pr_comment_id = $coq_pr_comment_id"
```

Comment:
Even nicer would be if at some point thats a single print routine which outputs something similar to json or OCaml record syntax.